cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityPrivilegesTest.scala

```diff
@@ -191,16 +244,16 @@ class SecurityPrivilegesTest extends DocumentingTest with QueryStatisticsTestSup
           resultTable()
         }
 
-        p("Like other `SHOW` commands, the output can also be processed using `YIELD` / `WHERE` / `RETURN`.")
+        p("Also here can the output be processed using `YIELD` / `WHERE` / `RETURN`.")
         query("SHOW ROLE architect PRIVILEGES AS COMMANDS WHERE command CONTAINS 'MATCH'", assertPrivilegeShown(Seq(
           Map("command" -> "GRANT MATCH {*} ON GRAPH * NODE * TO `architect`"),
           Map("command" -> "GRANT MATCH {*} ON GRAPH * RELATIONSHIP * TO `architect`"),
         ))) {
           resultTable()
         }
 
-        p("It is also possible to have privileges output as revoke commands. " +
-          "For more on revoke commands, please see <<administration-security-subgraph-revoke, The REVOKE command>>.")
+        p("""Once again is it possible to get the privilege commands formatted for revoking instead of granting or denying the privileges.
```

Comment:
```suggestion
        p("""Again, is it possible to get the privilege commands formatted for revoking instead of granting or denying the privileges.
```